lib/loggly/common.js

```diff
@@ -157,18 +167,25 @@ common.loggly = function () {
   }
   if (requestBody) {
     requestOptions.body = requestBody;
+    arrRetryLogs = arrRetryLogs.concat(requestBody);
   }
   function sendLogs() {
+    if (arrRetryLogs.length && !requestBody) requestOptions.body = arrRetryLogs[0];
     try {
       request(requestOptions, function (err, res, body) {
-        if (err) {
-          return onError(err);
-        }
-        var statusCode = res.statusCode.toString();
+        if (err) return onError(err);
+        statusCode = res.statusCode.toString();
         if(statusCode === '403') isValidToken = false;
+        if (statusCode === '500' || statusCode === '503' || statusCode === '504') retryOnServerError(res);
```

Comment:
@Shwetajain148 you should use the curl braces after if condition don't put under if statements without having {

Reply:
@mchaudhary Thanks for pointing out this. I will take of your input in my future commits.